modules/mig/metadata.yaml

```diff
@@ -80,61 +80,83 @@ spec:
         location: examples/compute_instance/tags
   interfaces:
     variables:
-      - name: autoscaler_name
-        description: Autoscaler name. When variable is empty, name will be derived from var.hostname.
+      - name: project_id
+        description: The GCP project ID
+        varType: string
+      - name: hostname
+        description: Hostname prefix for instances
+        varType: string
+        defaultValue: default
+      - name: mig_name
+        description: Managed instance group name. When variable is empty, name will be derived from var.hostname.
         varType: string
         defaultValue: ""
-      - name: autoscaling_cpu
-        description: Autoscaling, cpu utilization policy block as single element array. https://www.terraform.io/docs/providers/google/r/compute_autoscaler#cpu_utilization
-        varType: |-
-          list(object({
-              target            = number
-              predictive_method = string
-            }))
-        defaultValue: []
-      - name: autoscaling_enabled
-        description: Creates an autoscaler for the managed instance group
+      - name: region
+        description: The GCP region where the managed instance group resides.
```

Comment:
nit: The Google Cloud region

Reply:
Sure - These are auto-generated, so we'll need to update the source template.

Reply:
Done